v-next/hardhat/src/internal/cli/telemetry/sentry/init.ts

```diff
@@ -30,6 +30,7 @@ import { createGetModuleFromFilename } from "./vendor/utils/module.js";
 interface InitOptions {
   dsn: string;
   transport: (transportOptions: BaseTransportOptions) => Transport;
+  release?: string;
```

Comment:
It is optional here to allow the subprocess version to be booted without this information.